src/runtime_src/core/common/config_reader.h

```diff
@@ -793,11 +793,11 @@ get_flag_sw_emu_kernel_debug()
 }
 
 // This flag is added to exit device offline status check loop forcibly.
-// By default, device offline status loop runs for 120 seconds.
+// By default, device offline status loop runs for 320 seconds.
 inline unsigned int
 get_device_offline_timer()
 {
-  static unsigned int value = detail::get_uint_value("Runtime.dev_offline_timer", 120);
+  static unsigned int value = detail::get_uint_value("Runtime.dev_offline_timer", 320);
```

Comment:
is this timeout value related to the timeout in xgq.c? or this value is related to all subdevices probing time?

Reply:
It is related to all subdevices probing time during reset. Since, the waiting time in XGQ driver increases, we shall increase the timer value here as well.